src/client/src/rt-system/__tests__/connectionStream.test.ts

```diff
@@ -18,184 +19,51 @@ describe('createConnection$', () => {
 
   it('returns a cold observable', () => {
     // arrange
-    // TODO: remove this 'any' horror. It was done to remove type incompatibility errors. Sorry :(
-    const mockConnection: AutobahnConnection = (new MockAutobahnConnection() as any) as AutobahnConnection
+    const mockConnection: WsConnection = new MockWsConnection()
 
     // act
-    createConnection$(mockConnection)
+    connectionStream$(mockConnection)
 
     // assert
     expect(mockConnection.open).toHaveBeenCalledTimes(0)
     expect(mockConnection.close).toHaveBeenCalledTimes(0)
     expect(mockConnection.onopen).toHaveBeenCalledTimes(0)
     expect(mockConnection.onclose).toHaveBeenCalledTimes(0)
   })
-
```

Comment:
Do we need equivalents for those tests or did they only apply to Autobahn?

Reply:
Well spotted, I need to do something about this test.
With stomp, it works differently than Autobahn which provided an AutobahnSession class.
Now, we don't have a session with stomp, we just activate an endpoint with the broker address and port and we make rpc requests or rpc stream requests to this endpoint.